github/pulls.go

```diff
@@ -275,6 +277,9 @@ func (s *PullRequestsService) Merge(owner string, repo string, number int, commi
 	if options != nil {
 		pullRequestBody.CommitTitle = &options.CommitTitle
 		pullRequestBody.MergeMethod = &options.MergeMethod
+		if len(options.SHA) != 0 {
```

Comment:
Style suggestion, I think writing this as `if options.SHA != "" {` would be more readable. See my rationale [here](https://dmitri.shuralyov.com/idiomatic-go#empty-string-check). Feel free to change it if you agree.